src/Sylius/Bundle/ApiBundle/DependencyInjection/SyliusApiExtension.php

```diff
@@ -61,5 +64,12 @@ public function prepend(ContainerBuilder $container)
                 'client_manager' => 'sylius.oauth_server.client_manager',
             ],
         ]);
+
+        $container->setDefinition('fos_oauth_server.controller.token', new Definition(
+            TokenController::class, [
+                new Reference('fos_oauth_server.server'),
+                new Reference('fos_rest.decoder_provider'),
+            ]
+        ));
```

Comment:
Let's start a flamewar of preferences!

```php
$container->setDefinition(
    'fos_oauth_server.controller.token', 
    new Definition(TokenController::class, [
        new Reference('fos_oauth_server.server'),
        new Reference('fos_rest.decoder_provider'),
    ])
);
```

```php
$container->setDefinition('fos_oauth_server.controller.token', new Definition(TokenController::class, [
    new Reference('fos_oauth_server.server'),
    new Reference('fos_rest.decoder_provider'),
]));
```

```php
$container->setDefinition('fos_oauth_server.controller.token', new Definition(
    TokenController::class, 
    [new Reference('fos_oauth_server.server'), new Reference('fos_rest.decoder_provider')]
));
```

```php
$container->setDefinition('fos_oauth_server.controller.token', new Definition(
    TokenController::class, 
    [
        new Reference('fos_oauth_server.server'),
        new Reference('fos_rest.decoder_provider'),
    ]
));
```

Any of them looks more familiar than proposed one 🌮